index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>Leaflet Map</title>
+    
+    <!-- External Stylesheets -->
+<link rel="stylesheet" href="https://unpkg.com/leaflet@1.0.3/dist/leaflet.css" />
+    
+</head>
+   <body>
+       <div id="map" style="width: 705px; height: 375px"></div>
+    <!-- Our web map and content will go here -->
+    
+    <!-- Add the Leaflet JavaScript library -->
+<script src="https://unpkg.com/leaflet@1.0.3/dist/leaflet.js"></script> 
+      <script>
+    // Create variable to hold map element, give initial settings to map
+    var map = L.map('map',{ center: [42.362432, -71.086086], zoom: 14}); 
+          
+          // Add OpenStreetMap tile layer to map element
+L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', { attribution: '© OpenStreetMap' }).addTo(map);
+
+          // Create point feature for Kendall Square T Station
+var myDataPoint = L.marker([42.362432, -71.086086]).addTo(map); 
+          
+          // Create line feature for Mass Ave Bridge, style and add to map
+var myDataLine = L.polyline([[42.357227, -71.092631], [42.351411, -71.089723]],
+    {color: 'red', weight: 10}).addTo(map); 
+          
+          
+          // Create area feature for MIT, style and add to map
+var myArea = L.polygon([[42.353770, -71.103606], [42.355447, -71.104475],
+    [42.362681, -71.089830], [42.361829, -71.079230]],
+    {color: 'blue', weight: 4}).addTo(map);
```

Comment:
To my understanding the changes to your code need only include the above portion. So why are committing this whole file in a single commit?